### ISSUE TITLE
MODELRUN fails to return error in case of wrong number of outputs

### ISSUE THREAD

**Issue by daron1337** [RedisAI/RedisAI#116]
Example:
If the model expects more than one output and AI.MODELRUN is executed with only one output key, redis server logs the error but does not propagate it to the client.
The task is frozen until user manually kills it.

```
import redis
import numpy as np
import torch

class MyModule(torch.jit.ScriptModule):
   def __init__(self):
       super(MyModule, self).__init__()

   @torch.jit.script_method
   def forward(self, a, b):
       return a + b, a * b

my_script_module = MyModule()

print(my_script_module(torch.rand(2), torch.rand(2)))
my_script_module.save(“foo.pt”)

def set_redis_models():
   redis_client = redis.StrictRedis()
   print(“Redis client connected successfully”)

   with open(‘foo.pt’, ‘rb’) as f:
       model_pb = f.read()

   print(‘setting model...’)
   redis_client.execute_command(‘AI.MODELSET’, ‘model_test_cpu’, ‘TORCH’, ‘CPU’, model_pb)
   print(‘model set correctly’)
   x = np.ones((1), dtype=np.float32)
   print(‘setting tensor...’)
   redis_client.execute_command(‘AI.TENSORSET’, ‘tensor_test’, ‘FLOAT’, ‘1’, ‘BLOB’, x.tobytes())
   print(‘tensor  set correctly’)
   print(‘running model...’)
   redis_client.execute_command(‘AI.MODELRUN’, ‘model_test_cpu’, ‘INPUTS’, ‘tensor_test’, ‘tensor_test’, ‘OUTPUTS’, ‘output_test’)
   print(‘model run correctly’)

set_redis_models()
```

**Comment by lantiga** [comment]
Thank you @daron1337 and @dnlcrl ! #117 should fix the issue.

**Comment by lantiga** [comment]
Solved since #117 was merged
